src/mail_room.py

```diff
@@ -0,0 +1,169 @@
+#  _*_ coding:utf-8 _*_
+"""A simple python program to keep track of names and donations."""
+import sys
+import io
+
+
+GIFTER_DICT = {}
+
+prompts = ["Would you like to 'Send a Thank you' or 'Create a Report'?", "Enter the donator name or press 'L' to see a list of previous names", "How much is the donation?"]
+email_text = "Dear {},\nThank you so much for you donation of ${}. You are a good person.\n"
+
+
+def main_controller():
+    """Control flow of program."""
+    begin = send_or_report()
+    if begin is True:
+        name = get_name()
+        check_dict(name, GIFTER_DICT)
+        ammt = get_ammt()
+        update_dict(name, ammt)
+        send_email(name, ammt)
+        main_controller()
+    elif begin is False:
+        to_print = create_report()
+        if to_print is False:
+            main_controller()
+        else:
+            print_report(to_print)
+            write_to_file(to_print)
+            main_controller()
+
+
+def prompt_usr(prompt):
+    """Return the user response to a prompt."""
+    try:
+        input = raw_input
+    except NameError:
+        pass
+    usr_answer = input('{} \n>>>>>> '.format(prompt))
+    return usr_answer.lower()
+
+
+def validator(usr_response):
+    """Check the user answer."""
+    usr_response = usr_response.lower()
+    if usr_response == "send a thank you":
+        return True
+    elif usr_response == "create a report":
+        return False
+    elif usr_response == "q":
+        sys.exit()
+    else:
+        return "error"
+
+
+def send_or_report():
+    """Send a thank you or create a report."""
+    usr_response = prompt_usr(prompts[0])
+    validate = validator(usr_response)
+    if validate is True:
+        return True
+    elif validate is False:
+        return False
+    elif validate == "error":
+        print("Please choose a valid option")
+        send_or_report()
+
+
+def get_name():
+    """Get gifter name from user."""
+    usr_response = prompt_usr(prompts[1])
+    usr_response = usr_response.lower()
+    if usr_response == "q":
+        sys.exit()
+    elif usr_response == "l":
+        name_list = create_list(GIFTER_DICT)
+        print name_list
```

Comment:
Using this print statement instead of a print function makes this code not compatible with Python3.5. You should be putting `from __future__ import print_function` at the top of your module, and then you can use the print function across both versions.